apps/backend/src/auth/StfcUserAuthorization.ts

```diff
@@ -211,14 +214,23 @@ export class StfcUserAuthorization extends UserAuthorization {
         (role, index) =>
           stfcRoles.findIndex((r) => r.name == role.name) === index
       );
+
       const requiredInstruments =
         this.getRequiredInstrumentForRole(uniqueRoles);
+
       const currentUserInstruments =
         await this.instrumentDataSource.getUserInstruments(userNumber);
+
+      // Don't remove instruments from ISIS Staff and User officers as these will be manually assigned
       this.autoAssignRemoveInstruments(
         userNumber,
         requiredInstruments,
-        currentUserInstruments
+        currentUserInstruments,
+        !uniqueRoles.find(
+          (role) =>
+            role.name === 'ISIS Instrument Scientist' ||
+            role.name === 'User Officer'

```

Comment:
If not practical for now, I feel like we should at least open an issue to move these strings to constants. They seem to be present in a fair few places.

Reply:
I'll open a issue for it